sentinel5dl/__main__.py

```diff
@@ -141,14 +141,25 @@ def main():
             Example: 2019-09-17T23:59:59.999Z'''
     )
 
+    parser.add_argument(
+        '--useCertifi',
```

Comment:
Camel case is very uncommon for command-line arguments. Usually, kebab-case is used in these cases, e.g.:

```
% python -m venv --help|grep ' --.*-'
…
  --system-site-packages
  --without-pip
```

That said, I didn't pay close attention earlier and we already have some snake_case parameters like `'--begin_ts'`. Still, my suggestion would be to go all kebab-case and I'll create a pull request for the other parameters.

Reply:
if kebab-case is the thing we schould use it. i can modify all argparse parameters into kebab-case along with use-certifi or you can do it for the others. its your choice

Reply:
ok i changed only use_certifi to meet the other parameters in snake_case. we schould change them all to kebab-case together

Reply:
There already is pull request #43 but I can simply update that if this gets merged first.